src/screens/discover/components/TopMoversSection.js

```diff
@@ -1,16 +1,16 @@
 import lang from 'i18n-js';
 import React, { useCallback, useMemo } from 'react';
 import { IS_TESTING } from 'react-native-dotenv';
-import { TopMoverCoinRow } from '../coin-row';
-import { initialChartExpandedStateSheetHeight } from '../expanded-state/asset/ChartExpandedState';
-import { Centered, Column, Flex } from '../layout';
-import { MarqueeList } from '../list';
-import { Text } from '../text';
-import EdgeFade from './EdgeFade';
+import { TopMoverCoinRow } from '@/components/coin-row';
```

Comment:
Nit: we should preferably keep import edits like these out of most PRs, since they aren't directly related to the purpose of the PR.

It's not a huge deal, but now in addition to reviewing your refactor, I'm adding nits for using `@rainbow-me` aliases 😛

Reply:
Agreed

Reply:
Will address things like this in separate PRs it's not a big deal to do it like that